akka-actor-typed-tests/src/test/scala/akka/actor/typed/BehaviorSpec.scala

```diff
@@ -607,7 +607,7 @@ class DeferredJavaBehaviorSpec extends ImmutableWithSignalJavaBehaviorSpec {
 class TapJavaBehaviorSpec extends ImmutableWithSignalJavaBehaviorSpec with Reuse with SignalSiphon {
   override def behavior(monitor: ActorRef[Event]): (Behavior[Command], Aux) = {
     val inbox = TestInbox[Either[Signal, Command]]("tapListener")
-    (JActor.tap(
+    (JBehaviors.tap(
```

Comment:
Are these changes related to the PR? Would be nice to have the separate otherwise.

Reply:
It's from a "rebase from hell" where Actor => Behaviors change conflicted with "everything",
the name change was not complete and this change makes it correct. Would like to avoid yet another dance to just rename those into this direction again in a separate PR

Reply:
I guess the question is more of why did you touch that unrelated file. ;) Of course, you'll run into a "rebase of hell" if a potentially long-running PR contains unrelated changes to one of the central testing files of akka typed...